ko/documentation/quickstart/4/index.md

```diff
@@ -64,7 +64,7 @@ end
 매개 변수입니다.
 
 여기서 일어나고 있는 것은 리스트의 각 원소가 `name` 변수에 넘겨져서, 그 `name`을
-가지고 `puts “Hello#\{name}!”` 코드가 실행되고 있는 거랍니다.
+가지고 `puts “Hello #{name}!”` 코드가 실행되고 있는 거랍니다.
```

Comment:
```suggestion
가지고 `puts "Hello #{name}!"` 코드가 실행되고 있는 거랍니다.
```

Reply:
Thank you for the valuable suggestion. I've made the modifications as you recommended.